### ISSUE TITLE
Accessibility Audit: The "Close error message" button on Sign In error message is read incorrectly by Screen Reader

### ISSUE THREAD

**Issue by jnowell** [CityOfDetroit/bloom#962]
Solved with https://github.com/bloom-housing/bloom/issues/2249

**ENVIRONMENT**
Mobile: OS | OS Ver | Assistive Technology
Pixel 3a XL | Android 12 | Talk Back

**ISSUE**
Turn on the Screen Reader, and go to the [Sign In](https://detroit-public-dev.netlify.app/sign-in) page. Try to log in with an incorrect username/password. Observe the error message and focus your cursor on the 'X'. The close icon is being incorrectly read as "Multiplication sign button" For the consistency across the app, the 'X' button should be read as "Close" button.

This can also be replicated on a Macbook Chrome by using Voiceover (Command + F5) and then use the 'Rotor' (Control + Option + U) to find the Headings category.

https://user-images.githubusercontent.com/394908/148607186-aca46822-6af2-4517-a2fb-bba74f0afde7.mp4

**Comment by slowbot** [comment]
@kathyccheng We are waiting until the alert box component updates from core make it to Detroit. @seanmalbert
